deployment/environment_min.yml

```diff
@@ -20,8 +20,8 @@ dependencies:
   - deepdish
   - fenics
   - kwant
-  - mkl-service
```

Comment:
We need the mkl-service package for pypardiso; it's what you need to do `import mkl` in python. Do we need the mkl package explicitly as well?